### PR TITLE
i1043_student_view_message

### LINKED ISSUE #1043: Provide a clearer error message when instructor tries to view MyLA using Canvas's Student View
## Describe your problem or feature you'd like added
MyLA does not support the Canvas Student View as an LTI.  Currently, when an instructor uses the Student View and then clicks on My Learning Analytics, they get a white screen with the error message {"lti_error": "Launch failed due to 'email'"}.

This will be confusing for instructors.

## Describe the solution you'd like
Provide a clearer error message: "Student view is not available for My Learning Analytics."

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #1043 

I am having the message shown with 'lti_error` along with it. @jennlove-um is fine with this

With this issue #1003, I might refactor showing error user facing vs configuration issues during local development

### CODE REVIEW

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- dashboard/lti_new.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/my-learning-analytics/pullRequest?prid=6089203&bid=19933722)

File: dashboard/lti_new.py [hunk 1]
```diff
@@ -121,6 +121,11 @@ def extract_launch_variables_for_tool_use(request, message_launch):
     canvas_course_long_id = canvas_id_to_incremented_id(course_id)
     canvas_user_id = custom_params['canvas_user_id']
     canvas_user_long_id = canvas_id_to_incremented_id(canvas_user_id)
+    if 'email' not in launch_data.keys():
+        logger.error('Possibility that LTI launch by Instructor/admin becoming Canvas Test Student')
+        error_message = 'Student view is not available for My Learning Analytics.'
```
Comment (jonespm) [inline dashboard/lti_new.py:126]: Should we say "not yet available" since there was some discussion about making this available?

Comment (jennlove-um) [reply dashboard/lti_new.py:126]: I think stick with "not available" for the moment so we don't promise something that we don't know for sure will happen.

**Comment by pushyamig** [comment]
@jonespm is this ready to be merged?

**Review by jonespm — APPROVED** [review]

**Pull request merged**